apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -143,14 +148,19 @@ def get_query_server_config(name='beeswax', connector=None):
           LOG.info("Setting up Hive with the following node {0}".format(znode))
           if zk.exists(znode):
             hiveservers = zk.get_children(znode)
-            server_to_use = 0 # if CONF.HIVE_SPREAD.get() randint(0, len(hiveservers)-1) else 0
-            cache.set("hiveserver2", json.dumps({"host": hiveservers[server_to_use].split(";")[0].split("=")[1].split(":")[0], "port": hiveservers[server_to_use].split(";")[0].split("=")[1].split(":")[1]}))
+            server_to_use = abs(hash(str(username))) % 10 % len(hiveservers)
+            if hiveservers[server_to_use].split(";")[0].split("=")[1].split(":")[0] == DBMS_OLDHS2_NAME_CACHE.get(username):
```

Comment:
> DBMS_OLDHS2_NAME_CACHE.get(username):

Where is `DBMS_OLDHS2_NAME_CACHE` defined?

Reply:
Sorry，I forget pull all codes